jbpm-event-emitters/jbpm-event-emitters-elasticsearch/src/main/java/org/jbpm/event/emitters/elasticsearch/DefaultESInstanceViewTransformer.java

```diff
@@ -33,6 +33,8 @@ public ESRequest index(InstanceView<?> view) {
 
     @Override
     public ESRequest update(InstanceView<?> view) {
-        return new ESRequest(index, view.getCompositeId(), type, "update", view);
+        // TODO: investigate how update requests in the ES bulk api need to be created:
+        // https://www.elastic.co/guide/en/elasticsearch/reference/master/docs-bulk.html#docs-bulk
```

Comment:
index is the right operation (it is shame on me that I did not realize it earlier) because we are always sending the whole snapshot, not just a partial update, so you can remove the comment

Reply:
ok, thanks for confirming